client/app/components/Feed/Feed.jsx

```diff
@@ -1,20 +1,40 @@
-var React = require('react');
-var Filter = require('./Filter/Filter.jsx');
-var Maker = require('./Maker/Maker.jsx');
-var Message = require('./Message/Message.jsx');
-var Search = require('./Search/Search.jsx');
+import React from 'react';
+import Filter from './Filter/Filter.jsx';
+import Maker from './Maker/Maker.jsx';
+import MessageContainer from './Message/Message.jsx';
+import Search from './Search/Search.jsx';
 
-var Feed = React.createClass({
-  render: function () {
-    return (
-      <div> Hello Super Sweet Feed
-        <Filter />
-        <Maker />
-        <Message />
-        <Search />
-      </div>
-    );
+import getMuiTheme from 'material-ui/styles/getMuiTheme';
+import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider';
+import RaisedButton from 'material-ui/RaisedButton';
+
+let Plurb = React.createClass({
```

Comment:
Why are we instantiating the class in this way? Is it in order to use the getInitialState function?

class ExampleComponent extends React.Component { constructor() { super(); this. _handleClick = this. _handleClick.bind(this); this.state = Store.getState(); }

It looks like maybe we can just store the state in this.state